components/Buttons/src/MDCButton.m

```diff
@@ -503,7 +503,6 @@ - (void)animateButtonToHeightForState:(UIControlState)state {
 #pragma mark - BackgroundColor
 
 - (void)setBackgroundColor:(nullable UIColor *)backgroundColor {
-  // Turns out swift ignores NS_UNAVAILABLE
```

Comment:
I feel like we still need an explanation here.
"We recommend using setBackgroundColor:forState: This method is just a connivence method for the normal state.

Or we can keep it permanently deprecated.